pkg/storage/chunk/client/baidubce/bos_storage_client.go

```diff
@@ -36,10 +37,10 @@ func init() {
 }
 
 type BOSStorageConfig struct {
```

Comment:
Nit: It might be worth also adding a check like we have for S3 redacting / parsing